src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
```

Comment:
You missed one behavior:
<img width="856" alt="image" src="https://github.com/mate-academy/jv-oop-advanced/assets/41166578/8525aebb-aabd-4898-b258-36b8ea266a07">
```suggestion
public abstract class Figure implements Drawable, AreaCalculator {
```

```
public interface AreaCalculator {
    double getArea();
}
```